dkg-gadget/src/db/offchain_storage.rs

```diff
@@ -72,12 +72,14 @@ where
 		&self,
 		session_id: SessionId,
 	) -> Result<Option<LocalKey<Secp256k1>>, DKGError> {
+		log::info!(target: "dkg", "Offchain Storage : Fetching local keys for session {:?}", session_id);
 		let db_key = keys::LocalKey::new(session_id);
 		let maybe_decrypted_bytes = self.load_and_decrypt(codec::Encode::encode(&db_key))?;
 		match maybe_decrypted_bytes {
 			Some(decrypted_bytes) => {
 				let local_key = serde_json::from_slice(&decrypted_bytes.0)
 					.map_err(|e| DKGError::CriticalError { reason: e.to_string() })?;
+				log::info!(target: "dkg", "Offchain Storage : Fetched local keys for session {:?}, Key : {:?}", session_id, local_key);
```

Comment:
Printing the full `local_key` is not a good option here, it contains a lot of other information that is not important, we only interested in the `public_key` field, and in hex format which is great for us to compare.